internal/util.mli

```diff
@@ -114,9 +108,17 @@ val create_env : string -> (string * string) list
 val capitale : string -> string
 val index_of_next_char : string -> int -> int
 
+(** Search file (template, image...) in gw. *)
+val open_gw_etc_file : string -> in_channel option
```

Comment:
`Search file (template, image...) in gw`: What is `gw`?

Reply:
`/gw` is the executable folder